### ISSUE TITLE
Support for intelmq-cb-mailgen

### ISSUE THREAD

**Issue by bernhardreiter** [certat/intelmq-webinput-csv#40]
It would be nice to support [https://github.com/Intevation/intelmq-mailgen-release](intelmq-cb-mailgen) setups. 

Mainly this means that the path of the outgoing information has to be steered.
This can be done by a combination of
 * adding additional input via the webfrontend
 * documenting how to add additional configuration (e.g. rule scripts or mailgen scripts).

**Comment by bernhardreiter** [comment]
## Technical ideas
### Level 1: manual configuration by administrators
* Test and fix ability of webinput-csv to run on intelmy-cb-mailgen installations (Version of intelmq. Bring in Flask. Integrate into webserver) may include packaging
* Pointing towards the documentation of writing rule scripts, mailgen template and mailgen scripts.
* Testing for results have to be done on a staging system.

Disadvantages: Needs an admin trained for the task. Does not hold records of what has been done (would need to be done externally to the system).

### Level 2: usable by abuse-operators 
 * Create an additional table in the eventdb, working title `inserts` at to be able to hold texts for a notification batch.
 * Extend webinput-cvs to be able to write to `inserts` and to add a special feedname so that
   mailgen can notice which text to use.
 * Extend mailgen to access `inserts` and make the specific entry available to the scripts.
 * Extend Fody to show details about the past `inserts` so it on record.
 * (Will still be tested on a stating system).

### Level 3: additional testing mode
 This is level 2 extended to have a testing mode within the system.

**Comment by bernhardreiter** [comment]
Starting with level 1 (but on an intelmq-cb-mailgen setup on an old Ubuntu 16.04 LTS).

Other ideas include to partially rewrite, e.g. the backend or the frontend.

**Comment by wagner-certat** [comment]
> Starting with level 1 (but on an intelmq-cb-mailgen setup on an old Ubuntu 16.04 LTS).

That's waste of resources. Ubuntu 16.04 has Python 3.5 which won't be supported by IntelMQ's next feature release 2.3.0

**Comment by bernhardreiter** [comment]
@wagner-certat I agree that it is suboptimal, this I've suggested to change that requirement.
